package BagConfig;

message Costs
{
	required uint32 id 		= 1;
	required uint32 price	= 2;
}

message BagCosts
{
	required uint32 init 	= 1;
	required uint32 max 	= 2;
	repeated Costs  costs	= 3;	
}

